DKPhotoGallery/Preview/ImagePreview/DKPhotoBaseImagePreviewVC.swift

```diff
@@ -71,15 +70,7 @@ open class DKPhotoBaseImagePreviewVC: DKPhotoBasePreviewVC {
             DispatchQueue.main.async(execute: {
                 switch status {
                 case .authorized:
-                    if let animatedImage = contentView.animatedImage {
-                        ALAssetsLibrary().writeImageData(toSavedPhotosAlbum: animatedImage.data, metadata: nil, completionBlock: { (newURL, error) in
-                            self.showImageSaveResult(with: error)
-                        })
-                    } else if let imageURL = self.item.imageURL, imageURL.isFileURL, let data = try? Data(contentsOf: imageURL) {
-                        ALAssetsLibrary().writeImageData(toSavedPhotosAlbum: data, metadata: [:], completionBlock: { (newURL, error) in
-                            self.showImageSaveResult(with: error)
-                        })
-                    } else if let image = contentView.image {
```

Comment:
I'm not sure if this is correct to remove all of it, so please check this carefully.